packages/go_router/example/lib/query_params.dart

```diff
@@ -41,55 +42,16 @@ class App extends StatelessWidget {
             HomeScreen(families: Families.data),
         routes: <GoRoute>[
           GoRoute(
-            path: 'family/:fid',
-            builder: (BuildContext context, GoRouterState state) =>
-                FamilyScreen(
-              family: Families.family(state.params['fid']!),
-            ),
-            routes: <GoRoute>[
-              GoRoute(
-                path: 'person/:pid',
-                builder: (BuildContext context, GoRouterState state) {
-                  final Family family = Families.family(state.params['fid']!);
-                  final Person person = family.person(state.params['pid']!);
-                  return PersonScreen(family: family, person: person);
-                },
-              ),
-            ],
-          ),
+              path: 'family/:id',
```

Comment:
Consider renaming this file to "parameters.dart" or "path_and_query_parameters.dart" since it's demonstrating path and query parameters